From 6152ec801deb5d8e943a13ca92e4169f14f25295 Mon Sep 17 00:00:00 2001
From: Sergey Shelukhin <sershe@apache.org>
Date: Mon, 2 May 2016 15:13:25 -0700
Subject: [PATCH 0874/1164] CDH-48263 : HIVE-11887 : some tests break the
 build on a shared machine, can break HiveQA
 (Sergey Shelukhin, reviewed by Ashutosh Chauhan)

Change-Id: I1a98d57191e3363cd29f5bc5817c5585b3c0ad79
---
 itests/pom.xml |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/itests/pom.xml b/itests/pom.xml
index 1bdb8f8..9cf4b46 100644
--- a/itests/pom.xml
+++ b/itests/pom.xml
@@ -103,9 +103,6 @@
                     exit 1
                   fi
                   cp -f $HIVE_ROOT/data/conf/spark/log4j.properties $BASE_DIR/spark/conf/
-                  sed '/package /d' ${basedir}/${hive.path.to.root}/contrib/src/java/org/apache/hadoop/hive/contrib/udf/example/UDFExampleAdd.java > /tmp/UDFExampleAdd.java
-                  javac -cp  ${settings.localRepository}/org/apache/hive/hive-exec/${project.version}/hive-exec-${project.version}.jar /tmp/UDFExampleAdd.java -d /tmp
-                  jar -cf /tmp/udfexampleadd-1.0.jar -C /tmp UDFExampleAdd.class
                 </echo>
               </target>
             </configuration>
-- 
1.7.9.5

